stdlib/LinearAlgebra/src/eigen.jl

```diff
@@ -45,16 +48,22 @@ julia> vals, vecs = F; # destructuring via iteration
 
 julia> vals == F.values && vecs == F.vectors
 true
+
+julia> Matrix(exp(F)) ≈ exp(A)
+true
 ```
 """
 struct Eigen{T,V,S<:AbstractMatrix,U<:AbstractVector} <: Factorization{T}
     values::U
     vectors::S
-    Eigen{T,V,S,U}(values::AbstractVector{V}, vectors::AbstractMatrix{T}) where {T,V,S,U} =
-        new(values, vectors)
+    are_vectors_unitary::Bool
```

Comment:
This new field will not be required with #48657, which goes a step further in storing the left eigenvectors. For the unitary case `F.vectors === F.vectorsl` will have  th the same information.